subsys/bluetooth/mesh/Kconfig

```diff
@@ -513,6 +522,16 @@ config BT_MESH_RELAY_RETRANSMIT_INTERVAL
 	  messages, in milliseconds. Can be changed through runtime
 	  configuration.
 
+config BT_MESH_RELAY_SIMULTANEITY_COUNT
+	int "Maximum of simultaneous relay message support"
+	depends on BT_MESH_ADV_EXT_MULTI
+	depends on BT_EXT_ADV_MAX_ADV_SET > 1
```

Comment:
any particular reason this has a dependency to `BT_EXT_ADV_MAX_ADV_SET > 1`.

In the range below, if `BT_EXT_ADV_MAX_ADV_SET == 1` it will be:
```
	range 1 1
```
which is perfectly valid range,  where only `1` is a valid value.